mappings/net/minecraft/server/network/ServerPlayerEntity.mapping

```diff
@@ -164,10 +171,10 @@ CLASS net/minecraft/class_3222 net/minecraft/server/network/ServerPlayerEntity
 		ARG 1 tag
 	METHOD method_32749 writeGameModeToNbt (Lnet/minecraft/class_2487;)V
 		ARG 1 tag
-	METHOD method_33793 filtersText ()Z
+	METHOD method_33793 shouldFilterText ()Z
 	METHOD method_33794 acceptsMessage (Lnet/minecraft/class_2556;)Z
 		ARG 1 type
-	METHOD method_33795 shouldFilterText (Lnet/minecraft/class_3222;)Z
-		ARG 1 from
+	METHOD method_33795 shouldFilterMessagesTo (Lnet/minecraft/class_3222;)Z
```

Comment:
This parameter should be the from player

Reply:
I believe that's wrong, I'm pretty sure the method is whether you should filter messages sent *to* the specified player.

Reply:
Hmm, should this be named like `isFilteredBy` or `isMessageFilteredBy` to make it more obvious